reactions/src/HALO.cpp

```diff
@@ -247,17 +244,16 @@ static real sigma8d_integrandp_mgcamb(const PowerSpectrum& P, real R, real k) {
 }
 
 
-void HALO::scol_initp(double vars[]) const{
+void HALO::scol_initp(double vars[], bool mgcamb) const{
   SCOL scol;
 
   int loop_N = 30;
 
     // arrays to store values
   double myscolparams[3],lgmass[loop_N],sigar[loop_N],scolar0[loop_N],scolar1[loop_N],scolar2[loop_N];
 
-// select what variance to use  (if input P(k) is LCDM at z=0, vars[6]=0, if P(k) if modified at z=z, vars[6]=1)
 double sig1,sig2;
-  if (vars[6] < 0.5) {
+  if (!mgcamb) {
   // calculate sigma_8^2 and it's smoothing scale derivative @ R=8.
```

Comment:
useful to make clear what units you're working on, e.g. all k should be h/Mpc thus R should be Mpc/h, all input P(k) must be Mpc^3/h^3, etc.

Reply:
Agree. Since the same units are used throughout, we can probably add this to the readme file.